src/components/biggive-search/biggive-search.stories.ts

```diff
@@ -0,0 +1,16 @@
+export default {
+  title: 'Components/Demos',
+};
+
+const Template = args => `<biggive-search placeholderText="${args.placeholderText}" buttonText="${args.buttonText}" doSearch="${args.doSearch}"></biggive-search>`;
```

Comment:
@NoelLH this Storybook file has been troubling me quite a bit. I have _identically_ followed the guide you've linked to in the Readme file of this repository, but the args/props I pass to the stencil `biggive-search` component do not show up when viewing the component through storybooks. I haven't been able to figure it out. Basically, I can see them listed as Controls in the side/bottom bar, but they don't render on the component. If possible, could you please help with this? I would be very grateful!

Reply:
I've added more detail on Jira @alihejazi, but I think the key change needed is to used `dashed-case` attrs in HTML.

Reply:
Thanks!!